spec/models/refinery/jobs/job_application_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+require 'support/helper/generator_helper'
+
+module Refinery
+  module Jobs
+    describe JobApplication, type: :model do
+      before :each do
+        @job_application = FactoryGirl.create :job_application
+        @job = FactoryGirl.create :job
+      end
+
+      context 'validations' do
+        it 'should be functional' do
+          # Negative tests
+          expect(FactoryGirl.build(:job_application, name: nil)).not_to be_valid
+          expect(FactoryGirl.build(:job_application, phone: nil)).not_to be_valid
+          expect(FactoryGirl.build(:job_application, email: nil)).not_to be_valid
+          expect(FactoryGirl.build(:job_application, email: generate_string)).not_to be_valid
+          expect(FactoryGirl.build(:job_application, email: "#{generate_string}@")).not_to be_valid
+          expect(FactoryGirl.build(:job_application, email: "#{generate_string}@{generate_string}.")).not_to be_valid
+          expect(FactoryGirl.build(:job_application, cover_letter: nil)).not_to be_valid
+          expect(FactoryGirl.build(:job_application, resume: nil)).not_to be_valid
+
+          # Positive test
+          expect(FactoryGirl.build(:job_application, phone: '123-124-1234')).to be_valid
+        end
+      end
+
+      context 'destroy' do
+        it 'should be functional' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.